tests/reasoning/RuleEngine/DefaultForwardChainerCBUTest.cxxtest

```diff
@@ -50,7 +50,7 @@ void DefaultForwardChainerCBUTest::test_choose_rule(void)
 	logger().setPrintToStdoutFlag(true);
 	logger().setLevel(Logger::DEBUG);
 
-	JsonicControlPolicyParamLoader jscp(as, "test_cpolicy.json");
+	JsonicControlPolicyParamLoader jscp(as, "tests/reasoning/RuleEngine/test_cpolicy.json");
 	jscp.load_config();
```

Comment:
I mentioned this elsewhere, but .. no mater what I do, the unit est keeps failing at line 65 ... how are you able to run this test?